client/src/containers/User.jsx

```diff
@@ -100,7 +100,24 @@ class User extends Component {
     return (
       <div>
         <h2>{user}</h2>
-
+        {userPage.goal &&
```

Comment:
using prettier by any chance?

Reply:
Supposedly :/

Reply:
might want to run it here, small detail but it really feels off.

Reply:
Apparently Prettier likes it that way...